components/CalendarExportModal/CalendarExportModal.tsx

```diff
@@ -54,21 +53,40 @@ const CalendarExportModal = ({
       if (checkedShifts && checkedShifts.length > 0) {
         const checkedShiftsNames: { name: string; shift: string }[] =
           checkedShifts.map((shift) => {
-            return {
-              name: filters.find((f) => f.id === shift.id).name,
-              shift: shift.shift,
-            };
+            const name = maybeGetName(shift.id);
+
+            if (name != "") {
+              return {
+                name: maybeGetName(shift.id),
```

Comment:
Couldn't you just use the `name` variable here, instead of `maybeGetName(shift.id)`?